examples/regular_polygons.py

```diff
@@ -0,0 +1,89 @@
+import random
+import sys
+
+import geometry
+import pygame as pg
+
+FPS = 60
+RES = WIDTH, HEIGHT = 640, 480
+DARK_GREY = (10, 10, 10)
+SIDES = [3, 4, 5, 6, 7, 8, 9, 10, 11, 12]
+COLORS = {
+    3: (255, 0, 0),
+    4: (0, 255, 0),
+    5: (0, 0, 255),
+    6: (255, 255, 0),
+    7: (0, 255, 255),
+    8: (255, 0, 255),
+    9: (255, 255, 255),
+    10: (255, 127, 0),
+    11: (127, 0, 255),
+    12: (0, 255, 127),
+}
+
+
+class Game:
+    def __init__(self):
+        pg.init()
+        # pg.mouse.set_visible(False)
+        self.display = pg.display
+        self.screen = self.display.set_mode(RES)
```

Comment:
You could also combine these variables together deleting the variable creation of `RES`.